### ISSUE TITLE
[Request]: Direction options for Page Curl transition

### ISSUE THREAD

**Issue by adityakrshnn** [gl-transitions/gl-transitions#210]
Hi,

Can we get directions for page curl transition?

**Comment by MarcHbb** [comment]
Same request ! Like a reverse page curl transition ?

**Comment by scriptituk** [comment]
Use [SimplePageCurl](https://github.com/scriptituk/xfade-easing/blob/main/glsl/SimplePageCurl.glsl), it curls any direction and uncurls for reverse effect, see [description](https://github.com/scriptituk/xfade-easing/tree/main#transition-gl_simplepagecurl)

**Comment by verger-guo** [comment]
> Use [SimplePageCurl](https://github.com/scriptituk/xfade-easing/blob/main/glsl/SimplePageCurl.glsl), it curls any direction and uncurls for reverse effect, see [description](https://github.com/scriptituk/xfade-easing/tree/main#transition-gl_simplepagecurl)使用[SimplePageCurl](https://github.com/scriptituk/xfade-easing/blob/main/glsl/SimplePageCurl.glsl) ，它可以向任何方向卷曲并展开以获得相反的效果，请参阅[说明](https://github.com/scriptituk/xfade-easing/tree/main#transition-gl_simplepagecurl)

Is there an example of how to change the direction by touch point? Thanks

**Comment by scriptituk** [comment]
No, [SimplePageCurl.glsl](https://github.com/scriptituk/xfade-easing/blob/main/glsl/SimplePageCurl.glsl) is a GL Transition so it uses the API defined in the README of this repo.
The [original simple page curl effect here](https://www.shadertoy.com/view/ls3cDB) works by touch & drag.
The image below shows [SimplePageCurl.glsl](https://github.com/scriptituk/xfade-easing/blob/main/glsl/SimplePageCurl.glsl) in the [GL Transition Editor](https://gl-transitions.com/editor).
See the [documentation](https://github.com/scriptituk/xfade-easing#transition-gl_simplepagecurl).
![SimplePageCurl](https://github.com/user-attachments/assets/bc6e632a-7399-47ea-945c-764f6db7c16e)
